### PR TITLE
Add util to decode tests' name/id

### PR DESCRIPTION
**Pull request by Sae86**

### CODE REVIEW

**Review by dscott90 — APPROVED** [review]

File: chipsec/utilcmd/test_id_cmd.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        else:
+            module_id = generate_hash_id(self.module_name)
+        self.logger.log(f'Module ID is: {hex(module_id)}\n')
+
+    def get_name_from_id(self) -> None:
+        module_ids = get_module_ids_dictionary()
+        try:
+            module_name = list(module_ids.keys())[list(module_ids.values()).index(int(self.module_id, 16))]
+            self.logger.log(f'Module name is: {module_name}\n')
+        except:
```
[the remark this answers is not in the document]
Comment (npmitche) [reply chipsec/utilcmd/test_id_cmd.py:71]: Would probably recommend using this: 
```suggestion
            module_name = [k for k, v in module_ids.items() if v == int(self.module_id, 16)].pop()
            self.logger.log(f'Module name is: {module_name}\n')
        except IndexError:
```

File: chipsec/utilcmd/test_id_cmd.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# Contact information:
+# chipsec@intel.com
+#
+
+"""
+Generate a test ID using hashlib from the test's file name (no file extension).
+Hash is truncated to 28 bits. For test names use full path as seen in example below.
+
+Usage:
+    ``chipsec_util  name <test name>``
```
Comment (npmitche) [inline chipsec/utilcmd/test_id_cmd.py:26]: Needs to be:
```bash
chipsec_util id name <module name>
chipsec_util id hash <module hash>
```

File: chipsec/utilcmd/test_id_cmd.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    ``chipsec_util hash <test hash>``
+
+Examples:
+    >>> chipsec_main.py -m common.tools.generate_test_id name chipsec.modules.common.remap
```
Comment (npmitche) [inline chipsec/utilcmd/test_id_cmd.py:30]: These need to be updated too.

File: chipsec/utilcmd/test_id_cmd.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    >>> chipsec_main.py -m common.tools.generate_test_id hash 0x67eb58d
+"""
+
+from chipsec.library.returncode import get_module_ids_dictionary, generate_hash_id
+from chipsec.command import BaseCommand, toLoad
+from argparse import ArgumentParser
+
+
+class TestIdCommand(BaseCommand):
```
Comment (npmitche) [inline chipsec/utilcmd/test_id_cmd.py:39]: Should probably be "ModuleIdCommand", when I saw "test", I thought this was a unit test. (Same with the filename)

**Review by dscott90 — DISMISSED** [review]

**Review by npmitche — DISMISSED** [review]

**Comment by npmitche** [comment]
In the File `chipsec\tests\software\test_main.py`, in `test_read_files_in_tool_folder`, add `'tools.generate_test_id'` into the `expected` list to pass the unit test.

**Review by npmitche — APPROVED** [review]

**Review by dscott90 — APPROVED** [review]

**Pull request merged**